tests/e2e/specs/info.cy.js

```diff
@@ -42,5 +42,37 @@ describe('Info View', () => {
       .click({ force: true })
       .get('.v-expansion-panel--active')
       .should('have.length', 2)
+      .and('contain', 'Live')
+
+      // Xtriggers should be ticked or unticked
+      // depending on fulfillment:
```

Comment:
This comment does not match the test, which is only checking the xtrigger names are in the table.
```suggestion
```